### DISCUSSION TITLE
新聞分享:《台積電為什麼神？》：「AI行銷及報價系統」成為台積電與競爭對手拉大優勢的另一利器

### DISCUSSION THREAD

**Discussion by Jay-722** [1101-GS3073-C/Python-Community-GS3073-C#62] [科技新知]
AI已經融入在各個產業當中，一個產業具有獨創性就可以自己決定價格，且占據主要市場，但是現在市因次就需要客製化，客製化定價自由度高，客製化定價自由度高，但定價需要技巧。台積電運用AI的大數據動態報價系統，運用演算法，建立一套不同客戶競爭力分析的完整動態資訊系統，行銷已非傳統產業撰寫行銷規劃或作有創意的廣告可比擬。簡言之，這就是21世紀人工智慧大數據時代的行銷系統。
全文內容:https://www.thenewslens.com/article/157197

**Comment by jim11261314** [comment]
定價問題真的是一個很專業的領域，要如何知道市場的供需，
並即時給出買賣雙方都能接受的價格，是一個很複雜的問題。
我認為AI處理數據並能優化定價模型的能力，能夠減少未來的交易成本。
